src/examples/Casa/README.md

```diff
@@ -59,13 +51,80 @@ zoneLayer.loadZones([
   {
     partnerCode: 490,
     zones: [
+       {
+        zoneCode: 163,
+        isSelected: true,
+        isClickable: true,
+      },
+      {
+        zoneCode: 164,
+        isSelected: true,
+        isClickable: true,
+      },
       {
         zoneCode: 120,
         isSelected: true,
         isClickable: true,
       },
+      {
+        zoneCode: 121,
+        isSelected: true,
+        isClickable: true,
+      },
+      {
+        zoneCode: 122,
+        isSelected: true,
+        isClickable: true,
+      },
+      {
+        zoneCode: 123,
+        isSelected: true,
+        isClickable: true,
+      },
+      {
+        zoneCode: 124,
+        isSelected: false,
+        isClickable: true,
+      },
+    ],
+  },
+  {
+    partnerCode: 446,
+    zones: [
       {
         zoneCode: 170,
+        isSelected: false,
+        isClickable: true,
```

Comment:
Is that wanted to specify isSelected and isClickable everywhere?

Reply:
This was set in the file provided the SBB, not necessary though, since we have default values